data/lang/equipment-core/en.json

```diff
@@ -287,46 +327,202 @@
+  },
+
+  "MISSILE_RACK_442": {
+    "description": "Name for a multiple-missile external missile rack. 'Hydri' is a brand name.",
+    "message": "LH-240 Hydri Missile Rack"
+  },
+  "MISSILE_RACK_442_DESCRIPTION": {
+    "description": "Equipment tooltip",
+    "message": "External missile rack with a capacity of 4 medium missiles."
+  },
+  "MISSILE_RACK_442_FLAVOURTEXT": {
+    "description": "Equipment description",
+    "message": "Ready to take on the big league? With this big rack from Hydri, you'll always be fully loaded to unleash fiery destruction."
+  },
+
+  "MISSILE_RACK_423": {
+    "description": "Name for a multiple-missile external missile rack. 'Hydri' is a brand name.",
+    "message": "LH-320 Hydri Missile Rack"
+  },
+  "MISSILE_RACK_423_DESCRIPTION": {
+    "description": "Equipment tooltip",
+    "message": "External missile rack with a capacity of 2 large missiles."
+  },
+  "MISSILE_RACK_423_FLAVOURTEXT": {
+    "description": "Equipment description",
+    "message": "Make sure to keep your distance after firing! Hydri has packed 2 large missiles into this rack for some truly spectacular booms."
+  },
+
+  "MISSILE_RACK_524": {
+    "description": "Name for a multiple-missile external missile rack. 'Hydri' is a brand name.",
+    "message": "LH-420 Hydri Missile Rack"
+  },
+  "MISSILE_RACK_524_DESCRIPTION": {
+    "description": "Equipment tooltip",
+    "message": "External missile rack with a capacity of 2 huge missiles."
+  },
+  "MISSILE_RACK_524_FLAVOURTEXT": {
+    "description": "Equipment description",
+    "message": "Space for 2 huge missiles. Even we've run out of superlatives for what you can do with these.. we just hope you packed your sunnies!"
+  },
+
+  "MISSILE_RACK_543": {
+    "description": "Name for a multiple-missile external missile rack. 'Hydri' is a brand name.",
+    "message": "LH-340 Hydri Missile Rack"
+  },
+  "MISSILE_RACK_543_DESCRIPTION": {
+    "description": "Equipment tooltip",
+    "message": "External missile rack with a capacity of 4 large missiles."
+  },
+  "MISSILE_RACK_543_FLAVOURTEXT": {
+    "description": "Equipment description",
+    "message": "First Hydri brought you the awesome LH-320, now we've doubled its capacity! Go ahead, and make your mark on any system that dares to defy your might."
+  },
+
   "MISSILE_RAIL_S1": {
     "description": "Name for a single-missile external launcher. 'Cnida' is a brand name.",
     "message": "Cnida-101 Missile Rail"
   },
+  "MISSILE_RAIL_S1_DESCRIPTION": {
+    "description": "Equipment description",
+    "message": "External launcher for a single small missile."
+  },
+  "MISSILE_RAIL_S1_FLAVOURTEXT": {
+    "description": "Equipment description",
+    "message": "A quality external missile rail for S1 missiles of all types, brought to you by Cnida."
```

Comment:
Hydri and Cnida were both meant to share the same naming scheme (Hydra -> Hydri, Cnidaria -> Cnida) - I would consider both lines of missile racks to be designed/owned by Hydri Defense (see [Notion](https://www.notion.so/bszlrd/Manufacturers-1268c82e283f8071ade7feedeac2e2ca)), but perhaps the Cnida line is produced by a subsidiary?

Reply:
I see; I misunderstood and thought they were different manufacturers, not different product lines by the same manufacturer. Hence the very different tone in descriptions.

From last night, you mentioned Norwegian Kongsberg as the missile manufacturer? So they're the brand, with Cnida and Hydri being product lines?

ie:
"Norwegian Kongsberg is proud to introduce the Cnida range of missile rails for a compact  mounting solution for all of your missile needs. The Cnida S1 supports all S1 missiles manufactured to ISAAEEE standards such as the ubiquitous R40 range of rockets."

Reply:
.. and I just realised Kongsberg is a real defence manufacturer, not an in-game one...  don't confuse me like that!  Lol

Reply:
> I see; I misunderstood and thought they were different manufacturers, not different product lines by the same manufacturer. Hence the very different tone in descriptions.

I am open to making them different manufacturers - though I'd likely want to have Cnida be the brand/product name rather than the manufacturer's name (as otherwise the product name becomes "102 Missile Rail"... which seems more like a street in Britain than a high-tech piece of military hardware). 

> From last night, you mentioned Norwegian Kongsberg as the missile manufacturer?

That would have been @impaktor rather than myself ;)

Reply:
> I am open to making them different manufacturers

No; I'll tweak the descriptions. That said, did you choose "Hydri Defense" based on my misinterpretation? Might be slightly clearer if the product name and manufacturer name were not the same.. unfortunately I am incredibly bad at making up names so no suggestions here.

Otherwise:
"The Hydri LH-542 missile rack, by Hydri Defense, ..." 

As for model-names, how about:
- Cnida - single-missile rails
- Hydra - dual-missile racks  (Renaming Hydri -> Hydra here would resolve the manufacturer/model ambiguity)
- Medusa - quad-missile racks

?

Reply:
Now that you mention the name, I am kind of thinking of "Medusa" for an unguided rocket launcher (20+ rockets per launcher), and perhaps "Gorgon" for a beam laser of some sort (or maybe a heavy mining laser).

The idea of the "Hydra" missile rack was meant for all of the multi-missile launchers to differentiate it from the direct rail->missile adapter (being a pun on the multiple-missiles-per-hardpoint thing) - though I'm fine with "Hydra" for 2-missile rails and "Medusa" for 4-missile rails.